src/SubStatus.h

```diff
@@ -45,21 +48,55 @@
 /// - TransactionSubsMgr::getFullStatus() always returns one of these objects with the std::optional<BlockHeight> as
 ///   the active value, that is, blockHeight() will always be a valid pointer (even if it itself !has_value()).
 ///
+/// - ScriptHashTransactionSubsMgr::getFullStatus() always returns one of these objects with the std::optional<HashSet> as
+///   the active value, that is, hashSet() will always be a valid pointer (even if it itself !has_value()).
+///
+
+using HashSet = std::set<HashX>;
+
+namespace {
+    // assumption: `hashSet` is not empty!
+    // set is ordered to prevent different outcome of the same element composition
+    inline QByteArray HashSetHash(const HashSet &hashSet) {
+//        static_assert (sizeof(decltype(hashSet.front())) <= 4, "Assumption below is for at most 32-bit heights");
```

Comment:
I am not sure how to ensure this assertion. I have commented it out.
Would need your help to restore it.